src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditHearingSummary.java

```diff
@@ -93,9 +92,6 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToStart(CaseDetails<Ca
         }
         caseData.getCicCase().setHearingSummaryList(hearingService.getCompletedHearingDynamicList(caseData));
 
-        DynamicList judicialUsersDynamicList = judicialService.getAllUsers(caseData);
```

Comment:
Removed because it causes the judges list to not be populated with previous value.